CHANGELOG.rst

```diff
@@ -16,6 +16,8 @@ Changed
 Fixed
 ~~~~~
 
+- Fix type annotation for `decode()`/`encode()`'s parameter key `#605 <https://github.com/jpadilla/pyjwt/pull/605>`__
```

Comment:
I think this could be more descriptive about what the change is. "Fix"  doesn't really describe what the user should exepct.